### PR TITLE
Adds a "dismember" butchery option that's very fast but yields very little

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
```SUMMARY: Features "Corpses can now be dismembered as a butchery option, which takes very little time."```

#### Purpose of change
This is mainly intended for acid-blooded zombies, which need to be butchered instead of pulped at the moment. The only way around this is to go through a lot of mutation to get Acidproof, and I thought it'd be interesting to allow another route. Quick butchering still cares about yields, so it's comparatively slow; if you wanted to destroy a corpse and didn't care about its yield, it would be much faster.

#### Describe the solution
Adds a Dismember option to the butchery menu. This takes very little time (between 30-40 seconds, though it can't be faster than 30 seconds) and yields very little flesh.

Dismembering takes 10% of normal butchery time (where quick butchery is 100% and full butchery is 600%) and yields no harvest except a tiny amount of flesh.

#### Describe alternatives you've considered
* Originally this PR allowed decapitation for characters with bladed weapons and decent survival, which took two turns. This was deemed unfriendly to canon, so I changed it to its current form.

### CODE REVIEW

**Comment by kevingranade** [comment]
> A butchery option called "dismember" that served only to hack apart the body with no regard to its meat and took very little time.

I much prefer this option.  It's not canonical that beheading will automatically stop blob regeneration, but it is canonical that dismemberment will.

**Comment by Xhuis** [comment]
Changed it to dismembering, which makes it reliant on butchery tool. Dismembering takes 10% of normal butchery time (a quick butchery usually takes on average 6 minutes, where dismembering usually takes about 36 seconds) and can't be slower than 30 seconds. It yields very little flesh, but destroys the body.

File: src/activity_handlers.cpp [hunk 1]
```diff
@@ -1101,6 +1124,19 @@ void activity_handlers::butcher_finish( player_activity *act, player *p )
             }
             corpse_item.set_flag( "SKINNED" );
             break;
+        case DISMEMBER:
+            switch( rng( 1, 3 ) ) {
+                case 1:
+                    p->add_msg_if_player( m_good, _( "You hack the %s apart." ), corpse_item.tname().c_str() );
```
Comment (Night-Pryanik) [inline src/activity_handlers.cpp]: You can remove the `.c_str()` part from all the messages.

**Comment by Xhuis** [comment]
🔫

**Comment by FantasticFwoosh** [comment]
How does this differentiate with fieldprep + quartering, if the process to the latter is skipped?

**Comment by Xhuis** [comment]
You can't quarter without field prepping, meaning there's no reason to assume the corpse isn't first field prepped. Quartering would be a valid solution, but the field prepping requirement means it's not. Quartering is also meant to preserve the flesh, and dismembering isn't.

**Comment by Shodan14** [comment]
Just for comparison, how long does a pulp take?

**Comment by Xhuis** [comment]
Pulping depends on your weapon and the zombie. Usually 3-4 turns. Hulks take a few minutes. If you're acidproof, pulping is faster.

**Comment by Night-Pryanik** [comment]
Comment in this line
https://github.com/CleverRaven/Cataclysm-DDA/blob/60ad4a4a0e4140b0329e53c65d1e7309cb991a15/src/mondeath.cpp#L200-L201
states that pulping should make 4000 points of damage to the corpse.

**Comment by kevingranade** [comment]
Rebased and merged.

**Pull request closed without merging**